### PR TITLE
Add anchor points for references in the glossary (terms_and_definitions.adoc)

### PR DESCRIPTION
**Pull request by mettelilienthal**
## Describe your changes

add anchor points so that the glossary terms can be referenced within the standard

### CODE REVIEW

File: content/terms_and_definitions.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
+[#top-terms_and_definitions-terms_and_definitions]
+= Terms and definitions
+
+ifndef::include-only-once[]
+:root-path: ../
+include::{root-path}_config.adoc[]
+endif::[]
+
+3D asset::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:9]:
```suggestion
[[sec-3D-Asset]] 3D asset::
```

File: content/terms_and_definitions.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+The combination of an object's 3D geometry, assigned material properties, and
+specific metadata.
+
+3D model::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:13]:
```suggestion
[[sec-3D-model]] 3D model::
```

File: content/terms_and_definitions.adoc [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+A mathematical coordinate-based representation of an object's geometrical
+surface. Material properties used for rendering purposes can be part of the 3D model. This excludes physical material properties. For a combination of 3D
+geometry and physical material properties, see 3D asset.
+
+3D environment::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:18]:
```suggestion
[[sec-3D-environment]] 3D environment::
```

File: content/terms_and_definitions.adoc [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+A 3D representation of a simulated sensor's surroundings. It can consist of
+multiple 3D assets.
+
+Co-simulation::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:22]:
```suggestion
[[sec-Co-simulation]] Co-simulation::
```

File: content/terms_and_definitions.adoc [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+Simulation of multiple independent models connected with interfaces. A
+co-simulation platform handles the timing, synchronization, and data transfer between the
+individual models.
+
+Ground truth::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:27]:
```suggestion
[[sec-Ground-truth]] Ground truth::
```

File: content/terms_and_definitions.adoc [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+Defines true static and dynamic states of objects. The true states are either
+determined by the content creator or defined by an omniscient observer, for
+example, a simulation platform.
+
+Machine perception::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:32]:
```suggestion
[[sec-Machine-perception]] Machine perception::
```

File: content/terms_and_definitions.adoc [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+The ability of systems to interpret sensory data and to reliably detect objects
+and other road users relevant to the operation of automated driving. The data is
+used to generate situational awareness of the vehicle environment.
++
+Source: Klaus Dietmayer, Predicting of Machine Perception for Automated Driving, 2016
+
+Material::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:39]:
```suggestion
[[sec-Material]] Material::
```

File: content/terms_and_definitions.adoc [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+A collection of material properties belonging to one entity.
+
+Material assignment::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:42]:
```suggestion
[[sec-Material-assignment]] Material assignment::
```

File: content/terms_and_definitions.adoc [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+A mapping of material IDs to meshes of 3D models, or pixels of textures mapped to these meshes. The material ID is a unique
+pointer to a set of material properties in separate material files.
+
+Material classification::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:46]:
```suggestion
[[sec-Material-classification]] Material classification::
```

File: content/terms_and_definitions.adoc [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+Standardized classification scheme that sorts materials into classes and groups
+of classes, for example, based on their physical properties.
+
+Material parameter file::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:50]:
```suggestion
[[sec-Material-parameter-file]] Material parameter file::
```

File: content/terms_and_definitions.adoc [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+A file in JSON format containing defined material properties according to the
+OpenMATERIAL standard.
+
+Material property::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:54]:
```suggestion
[[sec-Material-property]] Material property::
```

File: content/terms_and_definitions.adoc [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+Denotes a physical, chemical, or geometric property of a material.
+
+Multispectral/hyperspectral material::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:57]:
```suggestion
[[sec-Multispectral_hyperspectral-material]] Multispectral/hyperspectral material::
```

File: content/terms_and_definitions.adoc [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+Material with wavelength-specific properties that are defined for multiple spectra. 
++
+Multispectral material shows up in imagery as 3-10 wider bands.
++
+Hyperspectral material shows hundreds of narrow bands. 
+
+Object::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:64]:
```suggestion
[[sec-Object]] Object::
```

File: content/terms_and_definitions.adoc [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+A tangible thing with defined 3D boundaries and discrete meshes. An object ID
+can be assigned.
+
+Object part::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:68]:
```suggestion
[[sec-Object-part]] Object part::
```

File: content/terms_and_definitions.adoc [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+A segment of an object that has distinct characteristics, for example, material,
+shape, or function.
++
+Example: A car door differs from the rest of the vehicle due to its function
+to open and close.
+
+Object type::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:75]:
```suggestion
[[sec-Object-type]] Object type::
```

File: content/terms_and_definitions.adoc [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+Class of objects with the same set of properties. Each instance of the class is
+described by the same properties, but has individual property values.
+
+Physical property::
+Denotes a characteristic of an object or material that can be measured.
+Examples: permittivity, density, index of refraction.
+
+Post rendering effect::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:83]:
```suggestion
[[sec-Post-rendering-effect]] Post rendering effect::
```

File: content/terms_and_definitions.adoc [hunk 17]
```diff
@@ -0,0 +1,117 @@
...
+Visual modification applied to graphics after initial rendering, enhancing
+appearance or realism without impairing performance.
+
+Scenario::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:87]:
```suggestion
[[sec-Scenario]] Scenario::
```

File: content/terms_and_definitions.adoc [hunk 18]
```diff
@@ -0,0 +1,117 @@
...
+A description of the behavior or temporal evolution of physical objects and
+environmental conditions on the driving infrastructure over an interval of time,
+including the movement of traffic participants or the change of environmental
+conditions.
++
+Source: ASAM OpenSCENARIO v1.3.0 standard, 2024
+
+Signal energy propagation::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:95]:
```suggestion
[[sec-Signal-energy-propagation]] Signal energy propagation::
```

File: content/terms_and_definitions.adoc [hunk 19]
```diff
@@ -0,0 +1,117 @@
...
+The act of a signal traveling through the environment and potentially
+interacting with objects and particles.
++
+Source: Linnhoff et al., Towards Serious Perception Sensor Simulation for Safety Validation of Automated Driving - A Collaborative Method to Specify Sensor Models, 2021
+
+Simulation platform::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:101]:
```suggestion
[[sec-Simulation-platform]] Simulation platform::
```

File: content/terms_and_definitions.adoc [hunk 20]
```diff
@@ -0,0 +1,117 @@
...
+Application that orchestrates and runs the simulations. It can be either an
+all-in-one solution including the generation of the 3D environment or a
+co-simulation platform orchestrating a co-simulation of multiple sensor models, vehicle dynamics models, and so on.
+
+Surface normal::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:106]:
```suggestion
[[sec-Surface-normal]] Surface normal::
```

File: content/terms_and_definitions.adoc [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+A vector that is perpendicular to a given object, surface, or geometrical
+primitive. It can be used to determine the face direction for 2D and
+3D rendering.
+
+Validation::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:111]:
```suggestion
[[sec-Validation]] Validation::
```

File: content/terms_and_definitions.adoc [hunk 22]
```diff
@@ -0,0 +1,117 @@
...
+A method for evaluating a simulation model's performance and accuracy by
+comparing simulated data to measured data and quantifying the differences.
+
+Vertex normal::
```
Comment (mettelilienthal) [inline content/terms_and_definitions.adoc:115]:
```suggestion
[[sec-Vertex-normal]] Vertex normal::
```

**Comment by mettelilienthal** [comment]
I've added anchor points for each glossary entry. The terms can now be referenced with the syntax `<<[anchor], [plain text]>>` . 

Referencing 'Machine perception', for example, would look like this:
`The <<sec-Machine-perception, machine perception>> includes...` .

In the text, it would show up as
The `machine perception` includes...
with a link to the respective glossary term.

**Comment by mettelilienthal** [comment]
@ClemensLinnhoff Please review :)

**Review by ClemensLinnhoff — APPROVED** [review]
I will merge this directly without a formal CCB review since there were no changes to the content and this PR is purely editorial. I hope this shortcut is okay @AsamDiegoSanchez.

**Review by ClemensLinnhoff — APPROVED** [review]
I will merge this directly without a formal CCB review since there were no changes to the content and this PR is purely editorial. I hope this shortcut is okay @AsamDiegoSanchez.

**Pull request merged**